Logger/FileLogger.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    class FileLogger : BaseLogger
+    {
+        private string path;
```

Comment:
Fields should be underscore PascalCase.
```suggestion
        private string _Path;
```